fuel-relayer/src/bin/testrun.rs

```diff
@@ -1,32 +1,80 @@
-use std::str::FromStr;
-
-use fuel_relayer::{Config, Service};
-
+use chrono::Utc;
```

Comment:
committing so much to chrono could be a long term problem. reconsider using `time`